src/Installers/Windows/AspNetCoreModule-Setup/Forwarders/build.proj

```diff
@@ -16,6 +16,7 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <Target Name="Build" DependsOnTargets="SetBuildDefaultEnvironmentVariables">
     <MakeDir Directories="$(ObjDir);$(BinDir)" />
-    <Exec Command="all.cmd $(Prompt) $(ObjDir) $(BinDir) $(Configuration)" />
+     <!-- Ignore link.exe ARM64X pure forwarder bug -->
+    <Exec Command="all.cmd $(Prompt) $(ObjDir) $(BinDir) $(Configuration)" IgnoreExitCode="true" />
```

Comment:
Is it possible to condition this just for arm64? So we don't mask errors in other builds?

Reply:
These forwarders are only built for Arm64